### PR TITLE
Esxi host exporter deployment

### PR DESCRIPTION
**Pull request by christopherhans**

### CODE REVIEW

**Review by viennaa — CHANGES_REQUESTED** [review]
Yay, new exporter on the run, thanks for driving this. Few things, before we can give it a spin.

File: prometheus-exporters/esxi-host-exporter/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        enabled: true
+        max_threads: 10
+        services:
+          - hostd
+          - nsx-opsagent
+          - nsx-proxy
+          - nsxa
+          - ntpd
+          - vpxa
+          - vvold
```
[the remark this answers is not in the document]
Comment (christopherhans) [reply prometheus-exporters/esxi-host-exporter/templates/configmap.yaml:22]: do I need a range? Coudn't I just write the list in the `values.yaml` and put here `services: {{ $.Values.collectors.critical_service_collector.services }}` ?

[the remark this answers is not in the document]
Comment (viennaa) [reply prometheus-exporters/esxi-host-exporter/templates/configmap.yaml:22]: AFAIK you do need a range.

[the remark this answers is not in the document]
Comment (christopherhans) [reply prometheus-exporters/esxi-host-exporter/templates/configmap.yaml]: Could you please check if latest version would be correct?

File: prometheus-exporters/esxi-host-exporter/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```
Comment (viennaa) [inline prometheus-exporters/esxi-host-exporter/templates/configmap.yaml:1]: please add `{{- if .Values.enabled }}` part in here too, the other components have it.

**Review by viennaa — APPROVED** [review]
Looks good, thanks 👍

**Pull request merged**